Wrappers/Python/test/test_functions.py

```diff
@@ -940,6 +943,43 @@ def proximal_conjugate_test(self, function, geom):
             np.testing.assert_allclose(a.as_array(), b.as_array(),
                                     rtol=1e-5, atol=1e-5)
 
+    def test_L1Norm_vs_WeightedL1Norm_noweight(self):
+        f1 = L1Norm()
+        f2 = WeightedL1Norm(weight=None)
+
+        assert f1.__class__.__name__ == 'L1Norm'
+        assert f2.__class__.__name__ == 'L1Norm'
+
+    def test_L1Norm_vs_WeightedL1Norm(self):    
```

Comment:
Can we also test the case when we have a scalar weight? To match with weightedL2NormSquared